.github/workflows/integration-tests.yml

```diff
@@ -3,28 +3,28 @@ name: integration-tests
 on:
   workflow_dispatch:
   push:
-    branches: [main]
-    paths-ignore:
-      [
-        "abstractions/**",
-        "authentication/**",
-        "serialization/**",
-        "http/**",
-        "**.md",
-        ".vscode/**",
-        "**.svg",
-      ]
-  pull_request:
-    paths-ignore:
-      [
-        "abstractions/**",
-        "authentication/**",
-        "serialization/**",
-        "http/**",
-        "**.md",
-        ".vscode/**",
-        "**.svg",
-      ]
+  #   branches: [main]
```

Comment:
can you expand on the comment out please? I think it's not triggering anymore because of the removal of pull request